examples/arduino/LVGL_Arduino/LVGL_Arduino.ino

```diff
@@ -67,6 +67,12 @@ void my_touchpad_read( lv_indev_t * indev, lv_indev_data_t * data )
      */
 }
 
+/*use Arduinos millis() as tick source*/
+uint32_t my_tick(void)
```

Comment:
```suggestion
static uint32_t my_tick(void)
```

Reply:
sorry, I haven't seen that I simply could commit your suggestion. I added the suggestion in my branch.